### DISCUSSION TITLE
How safe is to use persist data if it stores data in eg localStorage?

### DISCUSSION THREAD

**Discussion by arsekil** [pmndrs/zustand#2865] [Q&A]
Greetings,

I would like to implement a JWT token verification mechanism into my NextJS app, and my question is, if the user signs out from my app, i want to keep a sensitive info available for 24 hours between logins. 
I have read the docs and it can store data in localStorage. how safe is this taking in account the data needed to be persisted needs to be kept extremely safe.

Thanks in advance. Much appreciated.

**Comment by dbritto-dev** [comment]
@andrejmoltok the persist middleware by default use `localStorage` so I encourage you to read about that first
